user_scripts/wrangler_by_sample.sh

```diff
@@ -43,6 +55,7 @@ singularity_bindings="-B $project_resources:/opt/project_resources
  -B $output_folder:/opt/analysis
  -B $input_sample_sheet_directory:/opt/input_sample_sheet_directory
  -B $fastq_dir:/opt/data
+ -B /home/charlie/projects/MIPTools_wrangler_in_sif/snakemake:/opt/snakemake
```

Comment:
I'm hoping this path isn't hardcoded into the final shell script. Will look for a future commit that removes it